### PR TITLE
Add additional columns to kubectl output

### PR DESCRIPTION
**Pull request by JasonWhall**
**Reason for Change**:

Adds additional columns to kubectl output for AAD Pod Identity Custom Resources.

**Requirements**

- [x] squashed commits
- [x] included documentation
- [x] added unit tests and e2e tests (if applicable). See [test standard](https://github.com/Azure/aad-pod-identity/blob/master/CONTRIBUTING.md#test-standard) for more details.
- [x] ran `make precommit`

**Issue Fixed**:

- #829 
- #830 

**Please answer the following questions with yes/no**:

Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?

- [ ] yes
- [x] no

### CODE REVIEW

**Review by aramase — COMMENTED** [review]
@JasonWhall Thank you for adding this. Could you also add this to the CRD definitions in [manifest_staging/deploy/infra](https://github.com/Azure/aad-pod-identity/tree/master/manifest_staging/deploy/infra) folder?

**Review by chewong — COMMENTED** [review]
We use `kubebuilder` to generate CRDs Could you:
- check out https://book.kubebuilder.io/reference/generating-crd.html#additional-printer-columns on how to add additional columns via kubebuilder markers on the Go type
- `make generate-crds` to generate CRDs after making the changes

**Comment by JasonWhall** [comment]
@chewong @aramase I've updated the kubebuilder flags and hopefully correctly generated the crds and put the result in the right place! There's a bit of noise in the files, assuming this is ok.

Thanks, TIL ❤️

File: pkg/apis/aadpodidentity/v1/types.go [hunk 1]
```diff
@@ -31,6 +31,9 @@ const (
 
 // AzureIdentity is the specification of the identity data structure.
 //+k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+//+kubebuilder:printcolumn:name="Type",type="string",JSONPath=".spec.type",description="",priority=1
```
Comment (chewong) [inline pkg/apis/aadpodidentity/v1/types.go:34]: I am thinking maybe we can assign the priority to 0 so we don't have to append `-owide` during `kubectl get` to get the additional information. What do you think?

Comment (JasonWhall) [reply pkg/apis/aadpodidentity/v1/types.go]: Yep, makes sense to me as they should be present on all resources of this type 👍 I've updated the PR to reflect this.

**Review by chewong — APPROVED** [review]
This is great! Thanks for working on this

**Review by aramase — APPROVED** [review]
Thank you for PR!

lgtm

**Pull request merged**